.github/workflows/swap-ci-workflow.yml

```diff
@@ -103,7 +111,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: "LedgerHQ/app-exchange"
-          ref: "b3bf722138c19453bc9bef59e1b421cf0f5e8328"
+          ref: "ca99443823ae13b199f799b902207e78fa84f7c1"
```

Comment:
Do you want to checkout this particular ref or would `develop` or `master` do?

Reply:
Fixed 2aa3514152a93e616f8bccde589fbdae0770d68f